ssr-testing/next.config.js

```diff
@@ -1,19 +1,5 @@
-const path = require('path');
-const withTM = require('next-transpile-modules')([]);
-
-module.exports = withTM({
-  webpack(config) {
-    // https://github.com/josephluck/next-typescript-monorepo/blob/master/blog/next.config.js
-    const babelRule = config.module.rules.find((rule) =>
-      rule.use && Array.isArray(rule.use)
-        ? rule.use.find((u) => u.loader === 'next-babel-loader')
-        : rule.use.loader === 'next-babel-loader'
-    );
-
-    if (babelRule) {
-      babelRule.include.push(path.resolve('../'));
-    }
-
-    return config;
+module.exports = {
+  experimental: {
+    externalDir: true,
```

Comment:
Fills in for `next-transpile-modules`, is an experimental feature but in my testing seems to work great for our use case https://github.com/vercel/next.js/pull/22867